php-classes/Slate/CBL/CompetenciesRequestHandler.php

```diff
@@ -171,13 +171,21 @@ public static function handleDemonstrationsRequest(Competency $Competency)
 
         // query demonstrations sums
         try {
-            $skillDemonstrations = DB::allRecords(
-                'SELECT Demonstration.ID AS DemonstrationID, Demonstration.StudentID, DemonstrationSkill.SkillID, DemonstrationSkill.Level'
-                .' FROM (SELECT ID FROM `%s` WHERE CompetencyID = %u) AS Skill'
-                .' JOIN `%s` DemonstrationSkill'
-                .'  ON DemonstrationSkill.SkillID = Skill.ID'
-                .' JOIN (SELECT ID, StudentID FROM `%s` WHERE StudentID IN (%s)) Demonstration'
-                .'  ON Demonstration.ID = DemonstrationSkill.DemonstrationID'
+            $skillDemonstrations = DB::allRecords('
```

Comment:
I don't like using multiline strings for embedded code like SQL cause I like to keep indentation consistent within the PHP without filling the SQL with random whitespace. I wouldn't be mad if you used NOWDOCs for this though